helm/flink-kubernetes-operator/templates/webhook.yaml

```diff
@@ -85,7 +85,7 @@ metadata:
     cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/flink-operator-serving-cert
   name: flink-operator-{{ .Release.Namespace }}-webhook-configuration
 webhooks:
-- name: flinkoperator.flink.apache.org
+- name: validationwebhook.flink.apache.org
```

Comment:
Why the change?

Reply:
the webhook names need to be unique

Reply:
Do you mean the validating and the mutating webhook name clashes?

Reply:
If so, why not change both?

Reply:
will do